### ISSUE TITLE
keyboard is showing on back of the emoji view. is there any solution for this issue.

### ISSUE THREAD

**Issue by payaladbiz** [hani-momanii/SuperNova-Emoji#63]
![Screenshot_20210430-093139_HOW](https://user-images.githubusercontent.com/79822269/116647040-39cfaf00-a997-11eb-8b4a-6166c15f2aca.jpg)
Hi @hani-momanii I need a help as I am facing this weird issue. When I open the emojiview. the keyboard is still open at the back and its first row is visible at the back so it cause issue.
If you have any workaround to solve this then it will be a great help.

Kindly note this issue is only on 10+ android OS versions.
( I have checked with samsung galaxy M31s and RealMe 3 Pro)

**Comment by SumitSoni-001** [comment]
yes i am also facing the same issue.

**Comment by Mukhtarali6121** [comment]
did u get the solution?

**Comment by SumitSoni-001** [comment]
No bro

**Comment by varun7952** [comment]
@SumitSoni-001 it's still not fixed have you get any solution for this

**Comment by VirajS1202** [comment]
yes i am facing this issue any one get solution of this issue kindly inform me
